plugins/datahub-integration/src/main/resources/config/simplest-default.toml

```diff
@@ -0,0 +1,14 @@
+[global]
+geonetwork4_api_url = "/geonetwork/srv/api"
+datahub_url = "/datahub"
+proxy_path = ""
+
+[theme]
+primary_color = "#c82850"
+secondary_color = "#001638"
+main_color = "#555"
+background_color = "#fdfbff"
+
+[search]
+
```

Comment:
I would include the metadata_language option here

Reply:
I would also include a default translation for the title in fr + en